shopinvader/models/shopinvader_image_mixin.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ShopinvaderImageMixin(models.AbstractModel):
+    _name = 'shopinvader.image.mixin'
+
+    images = fields.Serialized(
```

Comment:
@sebastienbeau `images`is not really a list of images. It's a ist of images informations. I'm not convinced by the name... (a details)